### PR TITLE
Made sexy shaded box to add contrast to masthead

### PR DESCRIPTION
**Pull request by yefim**
sexy image

### CODE REVIEW

**Comment by Ceasar** [comment]
Looks good to me.

File: sass/pretty.sass [hunk 1]
```diff
@@ -58,11 +58,16 @@ $pennapps-red: #D80000
     width: 480px 
     text-shadow: 0 0 15px rgba(0, 0, 0, 0.6),0 -1px 1px rgba(0, 0, 0, 0.6)
     text-align: left
+    padding: 10px
+    border-radius: 10px
+    background-color: rgba(40, 40, 40, 0.7)
+    -moz-box-shadow:    0 0 20px rgba(20, 20, 20, 1)
+    -webkit-box-shadow: 0 0 20px rgba(20, 20, 20, 1)
+    box-shadow:         0 0 20px rgba(20, 20, 20, 1)
```
Comment (Ceasar) [inline sass/pretty.sass:66]: We should make a mixin for this.

Comment (Ceasar) [inline sass/pretty.sass:66]: Well, we can do it later when we need I suppose.

Comment (yefim) [inline sass/pretty.sass:66]: Sure, we can call it sexy-black-box or something

Comment (Ceasar) [inline sass/pretty.sass:66]: I winced a little after hitting merge and realizing I approved something that had the word 'sexy' in it.

Comment (yefim) [inline sass/pretty.sass:66]: You'll get used to it ;)

**Pull request merged**